spring-cloud-commons/src/main/java/org/springframework/cloud/client/loadbalancer/reactive/ReactorLoadBalancerClientAutoConfiguration.java

```diff
@@ -50,9 +51,10 @@
 	@Bean
 	public ReactorLoadBalancerExchangeFilterFunction loadBalancerExchangeFilterFunction(
 			ReactiveLoadBalancer.Factory<ServiceInstance> loadBalancerFactory, LoadBalancerProperties properties,
-			ObjectProvider<List<LoadBalancerClientRequestTransformer>> transformers) {
+			ObjectProvider<List<LoadBalancerClientRequestTransformer>> transformers,
+			LoadBalancerPropertiesFactory factoryProperties) {
```

Comment:
Please rename the argument to `propertiesFactory`.

Reply:
Changed